ramls/po_line_number.raml

```diff
@@ -1,14 +1,14 @@
 #%RAML 1.0
 title: "Orders Storage"
 baseUri: http://github.com/folio-org/mod-orders-storage
-version: v1
+version: v2
```

Comment:
hold off on this bump until release time

Reply:
Reverted